src/routes/index.js

```diff
@@ -116,6 +118,7 @@ export default function getRouteHandlers(
     'GET /tools/import/jobs/:jobId/progress': importController.getImportJobProgress,
     'POST /tools/import/jobs/:jobId/result': importController.getImportJobResult,
     'GET /tools/import/jobs/by-date-range/:startDate/:endDate/all-jobs': importController.getImportJobsByDateRange,
+    'POST /tools/import/assistant/prompt': assistantController.processImportAssistant,
```

Comment:
@blefebvre  The underlaying Open AI requests will be made to `/chat/completions`. Do we want to mirror the Open AI endpoints in Spacecat? Or is it better to stick with our own terminology?